#!/usr/bin/python
PACKAGE='adroc_diver'

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
gen.add("obs_timeout",    int_t,    0, "Timeout of observations", 1,  0, 10)
gen.add("bbox_target_ratio",    double_t,    0, "Ideal ratio of bouding box area to image area.", 0.7,  0., 1.)
gen.add("shoulder_target_ratio",    double_t,    0, "Ideal ratio of should distance to image width", 0.17,  0., 1.)

exit(gen.generate(PACKAGE, "diver_relative_position.py", "DRP_Params"))
